src/aiogram_dialog/widgets/kbd/request.py

```diff
@@ -54,3 +60,81 @@ async def _render_keyboard(
                 ),
             ],
         ]
+
+
+class RequestChat(Keyboard):
+    def __init__(
+        self,
+        text: Text,
+        RequestButton: KeyboardButtonRequestChat,
+        when: Union[str, Callable, None] = None,
+    ):
+        super().__init__(when=when)
+        self.text = text
+        self.RequestButton = RequestButton
+
+    async def _render_keyboard(
+        self,
+        data: Dict,
+        manager: DialogManager,
+    ) -> RawKeyboard:
+        return [
+            [
+                KeyboardButton(
+                    text=await self.text.render_text(data, manager),
+                    request_chat=self.RequestButton,
+                ),
+            ],
+        ]
+
+
+class RequestUser(Keyboard):
+    def __init__(
+        self,
+        text: Text,
+        RequestButton: KeyboardButtonRequestUser,
+        when: Union[str, Callable, None] = None,
+    ):
+        super().__init__(when=when)
+        self.text = text
+        self.RequestButton = RequestButton
+
+    async def _render_keyboard(
+        self,
+        data: Dict,
+        manager: DialogManager,
+    ) -> RawKeyboard:
+        return [
+            [
+                KeyboardButton(
+                    text=await self.text.render_text(data, manager),
+                    request_user=self.RequestButton,
```

Comment:
I am not sure it is good idea to have static request_id`

Reply:
What do you mean? If I'm not mistaking, there're no static request_id. We provide RequestButton above which was set up earlier by developer.

Reply:
I mean once you declare widget in dialog, it will have same request id here.